faculty.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'academic'
+
+class Faculty < Academic
+  attr_reader :salary, :target_raise
+
+  def initialize (options={})
+    super
+    @phase = 3
```

Comment:
I see ``` @phase ``` in a few spots, how can this move into a parent class and assign the appropriate value in the class?